app/controllers/settings/ldap_test_controller.rb

```diff
@@ -0,0 +1,136 @@
+require "net/ldap"
+require "json"
+require "openssl"
+require "base64"
+
+# LdapTestController is used to validate LDAP Connector before saving
+class Settings::LdapTestController < ApplicationController
+  def ldap_filter_search(ldap, baseDN, filter_string)
+    filter = Net::LDAP::Filter.construct(filter_string)
+    attrs = ["mail", "cn", "sn", "objectclass"]
+    search_object = ldap.search(
+      base:          baseDN,
+      filter:        filter,
+      attributes:    attrs,
+      return_result: true
+    )
+    search_object
+  end
+
+  def render_result(result_hash, status, message)
+    result_hash[:result] = { status: status, message: message }
+    render json: result_hash
```

Comment:
This needlessly returns the value of whatever the render function returns. Intended? If not the function should end in "return"

Reply:
Adding a return causes a rubocop "Style/RedundantReturn" error.